pkg/pods/pod.go

```diff
@@ -529,6 +529,12 @@ func (pod *Pod) setupConfig(manifest Manifest, launchables []launch.Launchable)
 	}
 
 	for _, launchable := range launchables {
+		// we need to remove any unset env vars
+		err = os.RemoveAll(launchable.EnvDir())
+		if err != nil && !os.IsNotExist(err) {
```

Comment:
I'll remove it since I agree it's not strictly necessary. By this point in the process, the parent must exist since it's already been installed at this point.